unittests/test_ipmi_tool.py

```diff
@@ -0,0 +1,94 @@
+#! /usr/bin/python3
+
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+import unittest
+import shutil
+from unittest.mock import Mock
+
+PROJECT_ROOT = "/".join(os.path.abspath(__file__).split("/")
+                        [:-2]) + "/low-level"
+os.sys.path.append(PROJECT_ROOT)
+
+from framework.utils.ipmi_client import IpmiFactory, Conf, store
+
+
+class TestIpmiTool(unittest.TestCase):
+
+    """Test IpmiTool over different platform and interface settings."""
```

Comment:
1 blank line is present still showing the error.

Reply:
This error is reported using pep257 tool.  [pydocstyle](https://pypi.org/project/pydocstyle/) supports most of the pep257. You can install and select pydocstyle linter in vscode. So this type of error you can directly get in vscode and you can try out different things, which can solve problem in your local. It is hard to do change and wait till codacy report and then again make change if your fix didn't solve the issue.